cli/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+    kotlin("jvm")
+    application
+    `java-test-fixtures`
+    id("io.gitlab.arturbosch.detekt")
+}
+
+application {
+    mainClass.set("ls.backup.cli.ApplicationKt")
+}
+
+dependencies {
+    implementation(project(":format"))
+    implementation(libs.kafka.clients)
+    implementation(libs.kotlinx.cli)
+    implementation(libs.s3.kotlin.client)
+    implementation(libs.apache.commons.compress)
+
+    runtimeOnly(libs.logger.nothing)
```

Comment:
why is this dependency required?

Reply:
I've added it because the kafka librarys want a logger implementation and warns if it has none which clutters the stdout.